.gitignore

```diff
@@ -1,3 +1,4 @@
 PublicWorkshops.xpr
 docs/Template/
 docs/
+.DS_Store
```

Comment:
was removing the LF here intentional?

Reply:
not on purpose, but not a breaker .. or is it?

Reply:
fix now in https://github.com/setheliot/aws-well-architected-labs/pull/12/commits/fe6255e90057d9f153ff6a7eb2e57f074c613698

Reply:
I did not know if it was a breaker... just noticed it.   
Dumb Git question: I see you added that commit to https://github.com/setheliot/aws-well-architected-labs/pull/12, but do you also need to add the other PRs?  If you do not won't the other PRs "undo"  https://github.com/setheliot/aws-well-architected-labs/pull/1?